resources/core/charts/service-catalog/charts/etcd/values.yaml

```diff
@@ -6,12 +6,14 @@ cluster:
       memory: 256Mi
     requests:
       memory: 32Mi
+  TLS:
```

Comment:
minor: why its uppercase?

Reply:
It's aligned with EtcdCluster CR definition https://github.com/kyma-project/kyma/pull/193/files#diff-f204c9017851295588b70289f10962f6R16